pytradfri/smart_task.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, gateway, raw):
         """Initialize the class."""
         super().__init__(raw)
         self._gateway = gateway
+        self.task_raw: TypeRawSimple = raw
```

Comment:
Instead of changing here, I suggest adding checks in ApiResource and keep the old code inside cases for the resources we don't want to update now (mood, group and smart task).

Reply:
Not sure I understood what you wanted me to not do here?

Reply:
Please revert these changes.